scarlet-protocol-stomp/src/main/java/com/tinder/scarlet/stomp/okhttp/support/StompHeaderAccessor.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * © 2018 Match Group, LLC.
+ */
+package com.tinder.scarlet.stomp.okhttp.support
+
+import com.tinder.scarlet.stomp.okhttp.models.StompHeader
+
+/**
+ * A base for classes providing strongly typed getters and setters as well as
+ * behavior around specific categories of headers (e.g. STOMP headers).
+ * Supports creating new headers, modifying existing headers,
+ * or copying and modifying existing headers.
+ */
+class StompHeaderAccessor private constructor(headers: Map<String, String>) {
+
+    companion object {
+
+        /**
+         * Custom header for the server to notify anything.
+         */
+        private const val STOMP_MESSAGE_HEADER = "message"
+
+        fun of(headers: Map<String, String> = emptyMap()): StompHeaderAccessor =
```

Comment:
Could we add `of` overload with headers `vararg` of pairs

Reply:
Added one more factory method